DataConnectors/Morphisec.json

```diff
@@ -0,0 +1,106 @@
+    "additionalRequirementBanner": "These queries and workbooks are dependent on Kusto functions based on Kusto to work as expected. Follow the steps to use the Kusto functions alias \"Morphisec\" \nin queries and workbooks. [Follow steps to get this Kusto function.](https://aka.ms/MorphisecFunction)",
+    "graphQueries": [{
+            "metricName": "Total data received",
+            "legend": "Morphisec",
+            "baseQuery": "\nMorphisec\n"
+        }
+    ],
+    "sampleQueries": [{
+            "description": "Threats count by host",
+            "query": "\nMorphisec\n\n            | summarize Times_Attacked=count() by SourceHostName"
+        }, {
+            "description": "Threats count by username",
+            "query": "\nMorphisec\n\n            | summarize Times_Attacked=count() by SourceUserName"
+        }, {
+            "description": "Threats with high severity",
+            "query": "\nMorphisec\n\n            | where toint( LogSeverity) > 7  | order by TimeGenerated"
+        }
+    ],
+    "dataTypes": [{
+            "name": "CommonSecurityLog (Morphisec)",
+            "lastDataReceivedQuery": "\nMorphisec\n\n            | summarize Time = max(TimeGenerated)\n            | where isnotempty(Time)"
+        }
+    ],
+    "connectivityCriterias": [{
+            "type": "IsConnectedQuery",
+            "value": [
+                "\nMorphisec\n\n            | summarize LastLogReceived = max(TimeGenerated)\n            | project IsConnected = LastLogReceived > ago(30d)"
+            ]
+        }
+    ],
+    "availability": {
+        "status": 1
+    },
+    "permissions": {
+        "resourceProvider": [{
+                "provider": "Microsoft.OperationalInsights/workspaces",
+                "permissionsDisplayText": "read and write permissions are required.",
+                "providerDisplayName": "Workspace",
+                "scope": "Workspace",
+                "requiredPermissions": {
+                    "read": true,
+                    "write": true,
+                    "delete": true
+                }
+            }, {
+                "provider": "Microsoft.OperationalInsights/workspaces/sharedKeys",
+                "permissionsDisplayText": "read permissions to shared keys for the workspace are required. [See the documentation to learn more about workspace keys](https://docs.microsoft.com/azure/azure-monitor/platform/agent-windows#obtain-workspace-id-and-key).",
+                "providerDisplayName": "Keys",
+                "scope": "Workspace",
+                "requiredPermissions": {
+                    "action": true
+                }
+            }
+        ]
+    },
+    "instructionSteps": [{"description":"These queries and workbooks are dependent on Kusto functions based on Kusto to work as expected. Follow the steps to use the Kusto functions alias \"Morphisec\" \nin queries and workbooks. [Follow steps to get this Kusto function.](https://aka.ms/MorphisecFunction)"},{
```

Comment:
Please place this kusto link in the json "https://aka.ms/sentinel-Morphiesc-parser"